README.md

```diff
@@ -2,8 +2,7 @@
 
 # Project Name
 
-> Description the project.
-
+> Bookstore is a book catalog where you can add and remove books, using react and redux.
 
 ## Built With
 
```

Comment:
- [ ] As pointed out by the  previous code reviewer, please make sure all the information in your `readme file` is related to you or your project. The `getting started` section still contains template information, please kindly update it to reflect your project as this will make your `readme file` more informative and professional.